airflow/api_connexion/openapi/v1.yaml

```diff
@@ -3579,7 +3579,7 @@ components:
             timezone:
               $ref: '#/components/schemas/Timezone'
             catchup:
-              type: boolean
+              type: string
```

Comment:
@pierrejeambrun, is this considered a breaking change?

Reply:
I feel we should still keep `True` and `False` as accepted values.

Reply:
DAGDetail is used as a response type for the endpoint `/dags/{dag_id}/details`, IMHO it's better to return one of the supported catchup values instead of a boolean even if the user uses a boolean in its dag.

Reply:
I believe so, changing the type of one field would be breaking. (Either in response or request)